src/core/Table/TableRowMemoized.tsx

```diff
@@ -70,6 +79,38 @@ const TableRow = <T extends Record<string, unknown>>(props: {
 
   const refs = useMergedRefs(rowRef, mergedProps.ref);
 
+  const subRowExpanderCellIndex = row.cells.findIndex(
+    (c) => c.column.id !== SELECTION_CELL_ID,
+  );
+
+  const getSubRowExpander = (cell: Cell<T>) =>
+    expanderCell ? (
+      expanderCell({
+        ...tableInstance,
+        ...{ cell, row: cell.row, value: cell.value, column: cell.column },
+      })
+    ) : (
+      <IconButton
+        style={{ marginLeft: cell.row.depth * 43, marginRight: 16 }}
```

Comment:
where does 43 come from?

Reply:
ok I see comment below. Do we really need it in two different places?

Reply:
Do you have a better idea? 😀 Previously we 4 were trying to manage to position everything correctly, so I was happy to make it work like this 😀

Reply:
Nice, used your suggestion!